packages/cli/test/integration/generators/relation.belongs-to.integration.js

```diff
@@ -20,24 +20,15 @@ const CONTROLLER_PATH = 'src/controllers';
 const REPOSITORY_APP_PATH = 'src/repositories';
 const sandbox = new TestSandbox(path.resolve(__dirname, '../.sandbox'));
 
-const sourceFileName = [
-  'order.model.ts',
-  'order-class.model.ts',
-  'order-class-type.model.ts',
-];
+const sourceFileName = ['order.model.ts', 'order-class.model.ts'];
```

Comment:
Would the tests work with only `order.model.ts`? Because it might help if we can remove all the extra fixtures and remove the `*class.model.ts` and `*class.type.model.ts` files (same for repos and controllers).

Reply:
I was keeping them for test in [test]( https://github.com/strongloop/loopback-next/pull/6350/files#diff-96c14786aa5559d4e191ecebb1731028R197). But it seems that those files are not needed in that test case. Will remove it 👍

Reply:
Ah sorry, I meant for all the tests for all relations, would it be possible? Could we narrow down the fixtures to remove the ones like `OrderClass` and `OrderClassType` and only use `Order`? But if not, I can approve this PR because the rest LGTM. 👍